modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/import/ImportModal.js

```diff
@@ -12,85 +12,63 @@
  * details.
  */
 
-import ClayButton from '@clayui/button';
-import ClayForm from '@clayui/form';
 import ClayModal from '@clayui/modal';
-import ClayProgressBar from '@clayui/progress-bar';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import {importStatus} from '../BatchPlannerImport';
-import Poller from '../Poller';
+import ImportPreviewModal from './ImportPreviewModal';
+import ImportProcessModal from './ImportProcessModal';
 
 const ImportModal = ({
 	closeModal,
+	dbFields,
+	fieldsSelections,
+	fileContent,
+	fileFields,
 	formDataQuerySelector,
 	formSubmitURL,
 	observer,
+	portletNamespace,
+	setFileContent,
+	setStartImport,
+	startImport,
 }) => {
-	const {errorMessage, loading, percentage} = Poller(
-		formDataQuerySelector,
-		formSubmitURL,
-		importStatus
-	);
-
-	return (
-		<ClayModal observer={observer} size="md">
-			<ClayModal.Header>
-				{Liferay.Language.get('import')}
-			</ClayModal.Header>
-
-			<ClayModal.Body>
-				<ClayForm.Group className={errorMessage ? 'has-error' : ''}>
-					<div className="progress-container">
-						<ClayProgressBar
-							value={percentage}
-							warn={!!errorMessage}
-						/>
-					</div>
+	const handleEditCell = (newValue, cellIndex, rowIndex) => {
```

Comment:
As I mentioned [here](https://github.com/liferay-commerce/liferay-portal/pull/2397#discussion_r827801659) please remove code related to partial features that anyway are unscoped.